ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePageVulnerabilities.tsx

```diff
@@ -296,6 +301,7 @@ function ImagePageVulnerabilities({ imageId, imageName }: ImagePageVulnerabiliti
                     onExceptionRequestSuccess={(exception) => {
                         selectedCves.clear();
                         showModal({ type: 'COMPLETION', exception });
+                        return onExceptionRequestSuccess();
```

Comment:
In `CVEsTableContainer`, I noticed that we have similar logic except the function we call is `refetchAll()` in that one. Here, we have `onExceptionRequestSuccess`. Does it make more sense to have this as `refetchAll()` as well? If you go through the logic, in words, for what happens on a successful exception request, we do the following:

1. Clear the selected CVEs
2. Show the completion modal
3. Refetch all

But this function call didn't seem as intuitive to read. Let me know your thoughts on that

Reply:
I agree, I think I originally went with `onExceptionRequestSuccess` so that it was general and we could add additional logic if needed. Since we aren't doing that though, I think readability wins and we can refactor later if/when this changes.